build_tools/cmake/build_e2e_test_artifacts.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
This is copied and modified from `build_tools/cmake/build_benchmarks.sh`. We will remove `build_tools/cmake/build_benchmarks.sh` once everything is migrated.

Reply:
Maybe put that comment in the file itself :-)

Reply:
Done.